.travis.yml

```diff
@@ -31,12 +44,6 @@ before_install:
 # the `install` step will run `bundle install` by default.
 script:
   - .ci/build.sh
-  # https://github.com/coala/coala-bears/issues/1037
```

Comment:
Why has this section been removed?

Reply:
See  @kriti21's PR.

@sladyn98 	, i suggest you make your PR only about fixing the spacing.  Leave the other problem to kriti.

Or split your commit into two well written commits, as they are two distinct fixes.

Reply:
@jayvdb yep agreed I made these changes intially with the aim of having my PR merged but since @kriti21 got her CI green I am removing these changes :+1: